prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -1087,12 +1088,12 @@ class ActionHistory : Logging {
             fun createPropertiesByGroupingId(actionLogs: List<ActionLog>): GroupedProperties {
                 val itemsByGroupingId = mutableMapOf<String, MutableList<Int>>()
                 val messageByGroupingId = mutableMapOf<String, String>()
-                val scopesByGroupingId = mutableMapOf<String, ActionLog.ActionLogScope>()
+                val scopesByGroupingId = mutableMapOf<String, ActionLogScope>()
                 actionLogs.forEach { actionDetail ->
-                    val groupingId = actionDetail.detail.groupingId()
+                    val groupingId = actionDetail.detail.message
```

Comment:
Should keep grouping by `receiverName` for now

Reply:
Need help understanding this one